### ISSUE TITLE
When training the cub training set, the code always stops automatically every 13epoch

### ISSUE THREAD

**Issue by crzjustlearn** [zxhuang1698/interpretability-by-parts#10]
First, when training cub training set, the code always stops automatically every 13epoch, for example, 0-13, 14-27. I only changed the batchsize to 4, and at the same time, I changed the learning rate in the paper to 1e-3 (the learning rate in the code is 5e-4). I want to ask what's the matter, can't I change the batchsize?

The second problem is that after the training, the accuracy of the training set is 82.5, but in the visualization processing, only one of the 25 images is correctly judged. What's the problem.

Thank you very much.

**Comment by crzjustlearn** [comment]
There are also some questions. Why do we collect part and bounding box information in cub200, but they are not used in training? Is it really useful to use these two kinds of information? Another question is why we need to subtract one when setting label in cub200. After subtracting one, it does not correspond to the data set completely? I don't quite understand
